portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase

```diff
@@ -891,6 +891,49 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-126159. The new user can publish a content page without set permissions for custom field page."
+	@priority = "4"
+	test NewUserCanPublishWithoutCustomFieldPermissions {
+		task ("Add a custom field for Page") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Custom Fields");
+
+			CustomFields.addCP(
+				customFieldName = "Text Field",
+				customFieldType = "Input Field",
+				customInputDataType = "Text",
+				resourceName = "Page");
+		}
+
+		task ("Assign the site role to new user") {
+			JSONRole.assignRoleToUser(
+				roleTitle = "Administrator",
```

Comment:
The Administrator has Custom Fields permission. Create a role with permissions to add pages and contents.